src/core/constants.py

```diff
@@ -11,3 +11,4 @@ class CommonPaths:
 
     project_root = Path(__file__).parent.parent.resolve()
     resource_path = project_root.joinpath("resources")
+    log_path = project_root.joinpath("logs")
```

Comment:
@vasylboliuk it looks to me you could combine all these paths under some ProjectPath Enum class